governance-exercises/Le Bateleur - Poll Exercise 02/Course of Action.md

```diff
@@ -0,0 +1,61 @@
+
+The language in Atlas 2.6.6 is ambiguous. While "clear evidence" presents no issue, the phrase "significant suspicion" presents a gray area. Is "reasonable suspicion" equivalent to the "significant suspicion" threshold that we find in Atlas 2.6.6? There's no clear answer based on the text alone. But I am also aware of my [mandate](https://mips.makerdao.com/mips/details/MIP999#2-4-2-alignment-conservers-eligibility-requirements) to minimize risk to Maker and guard against misalignment threats. This is a close call, especially given the ambiguity in Atlas 2.6.6. I am compelled in this instance to err on the side of caution and determine that the materials provided by Diane *do* meet the specified threshold of "significant suspicion."
+
+But a determination from the Governance Facilitator is not to be the final word. Per [Atlas 2.6.6.1.1](https://mips.makerdao.com/mips/details/MIP101#2-6-6-1-1), a review of the decision by all FacilitatorDAOs is required. Because FacilitatorDAOs don't yet exist, running a governance poll can serve as a crucial "check" on the Governance Facilitator's power. There is also (outdated) [historical support](https://forum.makerdao.com/t/mip102c2-sp2-mip-amendment-subproposals/20016/19#h-1011-constitutional-boundaries-of-the-arbitration-scope-framework-67) for using a governance poll to adjudicate derecognition matters. (The former Maker Constitution, 10.11.3.4, states, "During the Pregame, all actions of derecognition must be decided through a MKR governance poll...")
+
+---
+
+### Should a whistleblower bounty be paid to Alice?
+
+This cannot be answered until the final determination is made as to whether Clive's identity has been compromised. [Atlas 2.6.6](https://mips.makerdao.com/mips/details/MIP101#2-6-6-aligned-delegate-privacy) states that a whistleblower bounty is paid when an actor has "provided useful information for determining that the privacy of an AD was compromised." (Arguably, there is a bit of ambiguity here, but not so much as to preclude a reasonable interpretation.) I interpret this to mean that a determination has to be made that the privacy of the Aligned Delegate was in fact compromised *before* a whistleblower is eligible to receive a bounty. 
+
+Therefore, Alice's bounty cannot be included in the coming Monday's poll concerning the breach-of-anonymity evidence against Clive. Should that poll determine the materials provided rise to the level of "significant suspicion", thus triggering Clive's derecognition, I would then do the necessary follow up with Alice on the bounty.
```

Comment:
I think you've landed on what I would view as the correct outcome but for different reasons. I think including the bounty in the poll introduces another consideration on voters beyond simply 'are the materials sufficient' it adds: 'do I think Alice should get the bounty' which has the potential to have a  different answer depending on the voter, and so should be separated.

Given the wording of the atlas, I don't think the bounty is a requirement, and remains at the discretion of the Governance Facilitator. So I would like to see governance facilitator considering the pros and cons of paying it in this instance.

In this scenario I would likely pay some proportion of the bounty <50% to given incentive to provide information, but disincentive for revealing it publicly prior to it being absolutely necessary.